__tests__/index.test.js

```diff
@@ -0,0 +1,56 @@
+const app = require('../app');
+const request = require('supertest');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const data = require('../db/data/test-data/index');
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe('GET /api/topics', () => {
+	test('GET:200 return an array of topics of the correct format', () => {
+		return request(app)
+			.get('/api/topics')
+			.expect(200)
+			.then(({ body }) => {
+				expect(body.topic.length).toBe(3);
```

Comment:
The topics should be send back on a key of 'topics' but currently they are on a key of 'topic'.